ts/Extensions/Annotations/Popup.ts

```diff
@@ -1229,7 +1217,7 @@ H.Popup.prototype = {
             return createElement(
                 DIV,
                 {
-                    className: PREFIX + 'tab-item-content'
+                    className: PREFIX + 'tab-item-content highcharts-no-mousewheel'// #12100
```

Comment:
The `PREFIX` variable is `highcharts-`. I think it would be better if you added those classes this way:
```js
className: PREFIX + 'tab-item-content' + ' ' + PREFIX + 'no-mousewheel'// #12100

```